src/vaults/AsyncRequests.sol

```diff
@@ -111,12 +125,9 @@ contract AsyncRequests is BaseInvestmentManager, IAsyncRequests {
         AsyncInvestmentState storage state = investments[vaultAddr][controller];
         require(state.pendingCancelDepositRequest != true, "AsyncRequests/cancellation-is-pending");
 
-        state.pendingDepositRequest = state.pendingDepositRequest + _assets;
-        VaultDetails memory vaultDetails = poolManager.vaultDetails(address(vault_));
-
-        sender.sendDepositRequest(
-            vault_.poolId(), vault_.trancheId(), controller.toBytes32(), vaultDetails.assetId, _assets
-        );
+        state.pendingDepositRequest += assets;
+        balanceSheet.escrow().pendingDepositIncrease(vaultDetails.asset, vaultDetails.tokenId, poolId, scId, assets);
```

Comment:
The escrow access can be improved but since we want to refactor the PerPoolEscrow anyways, I didn't put much effort into the current usage.